### PR TITLE
Adding Campaign Group template

### LINKED ISSUE #1894: Campaign Group template
Preprocess vars and create a template for the Campaign Group template.  

Ideally figure out ways to re-use as much code as possible.

### PR DESCRIPTION
**Pull request by blisteringherb**
- Adding a preliminary tpl for campaign group
  to allow bender work to begin.

Fixes #1894

@aaronschachter 
@DFurnes 
@barryclark

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc [hunk 1]
```diff
@@ -237,7 +167,7 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   // Collect all field_collection values as vars.
   $field_collections = array('faq', 'step_pre', 'step_post');
   foreach ($field_collections as $fc) {
-    $vars[$fc] = dosomething_campaign_get_field_collection_values($wrapper, 'field_' . $fc);
+    $vars[$fc] = dosomething_helper_get_field_collection_values($wrapper, 'field_' . $fc);
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc]: Shouldn't this be breaking things?  `dosomething_helper` vs `dosomething_helpers` ?

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc [hunk 2]
```diff
@@ -47,8 +47,8 @@ function dosomething_campaign_preprocess_node(&$vars) {
 
     if (!empty($vars['field_partners'])) {
       // Sets partners, sponsors, and partner_info arrays if present.
-      dosomething_campaign_preprocess_partners_vars($vars);
-      dosomething_campaign_format_partnerts_list($vars);
+      dosomething_helpers_preprocess_partners_vars($vars);
+      dosomething_helpers_format_partnerts_list($vars);
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc]: Lets fix this misspelling while we're in here. `partnerts`

Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc]: I see its spelled correctly below tho... wouldn't this have broken things too?  Please make sure to test campaigns with these changes to make sure all is good.

**Comment by aaronschachter** [comment]
If tested and confirmed for both campaigns and campaign groups, :+1:

**Comment by blisteringherb** [comment]
I screwed this up.  I'm going to have to close it and redo it.

**Pull request closed without merging**